From 79e2796df3609593ba745f9ea4228ec58535a993 Mon Sep 17 00:00:00 2001
From: Markus Armbruster <armbru@redhat.com>
Date: Mon, 11 Aug 2014 07:54:43 +0200
Subject: [PATCH 09/16] rhel: apic.version=0x11 for machine types <= 7.0

Message-id: <1407743689-13553-10-git-send-email-armbru@redhat.com>
Patchwork-id: 60513
O-Subject: [PATCH RHEV-7.1 qemu-kvm-rhev 09/15] rhel: apic.version=0x11 for machine types <= 7.0
Bugzilla: 1118665
RH-Acked-by: Eduardo Habkost <ehabkost@redhat.com>
RH-Acked-by: Miroslav Rezanina <mrezanin@redhat.com>
RH-Acked-by: Dr. David Alan Gilbert (git) <dgilbert@redhat.com>

Update RHEL compatibility properties for upstream commit aa93200.

Signed-off-by: Markus Armbruster <armbru@redhat.com>
---
 include/hw/i386/pc.h | 4 ++++
 1 file changed, 4 insertions(+)

Signed-off-by: Miroslav Rezanina <mrezanin@redhat.com>
---
 include/hw/i386/pc.h |    4 ++++
 1 files changed, 4 insertions(+), 0 deletions(-)

diff --git a/include/hw/i386/pc.h b/include/hw/i386/pc.h
index 99abd47..8be4cf4 100644
--- a/include/hw/i386/pc.h
+++ b/include/hw/i386/pc.h
@@ -533,6 +533,10 @@ bool e820_get_entry(int, uint32_t, uint64_t *, uint64_t *);
             .property = "memory-hotplug-support",\
             .value    = "off",\
         },{\
+            .driver   = "apic",\
+            .property = "version",\
+            .value    = stringify(0x11),\
+        },{\
             .driver   = "PIIX4_PM",\
             .property = "acpi-pci-hotplug-with-bridge-support",\
             .value    = "off",\
-- 
1.7.1

